### DISCUSSION TITLE
Defference with ESP NimBLE Host

### DISCUSSION THREAD

**Discussion by nexty-kondo** [h2zero/NimBLE-Arduino#717] [Q&A] [answered]
Hello.

I think NimBLE-Arduino is based on ESP NimBLE.
I want to know if there are difference between them about the part of host's Implementation Conformance Statement (ICS).

Thanks in advance.

**Comment by h2zero** [comment, marked as answer]
The only differences are that support for nRF devices has been maintained and esp-timer is not used, also a bug fix or 2.
